tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs

```diff
@@ -65,6 +85,77 @@ public void GraphicsExtension_Gd_IsInstalled(string imageTag)
             Assert.True((bool)((JValue)gdInfo.GetValue("PNG Support")).Value);
         }
 
+        [Theory]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
+        [InlineData("7.0")]
+        [InlineData("5.6")]
+        public async Task Check_If_Apache_IsConfigured_For_PHP(string imageTag)
```

Comment:
Rename the test to the pivot being tested i.e the casing of the file extension

Reply:
sure